boards/beagle/beaglebone_ai64/doc/index.rst

```diff
@@ -0,0 +1,112 @@
+.. _beaglebone_ai64:
+
+BeagleBone AI-64
+################
+
+Overview
+********
+
+BeagleBone AI-64 is a computational platform powered by TI J721E SoC, which is
+targeted for automotive applications.
+
+.. figure:: assets/bbai_64.webp
+   :align: center
+   :width: 600px
+   :alt: BeagleBoard.org BeagleBone AI-64
+
+Hardware
+********
+
+The BeagleBone AI-64 is powered by TI J721E SoC, which has three domains (Main,
```

Comment:
Branding thing.... Don't put "The" before "BeagleBone AI-64". See https://www.beagleboard.org/brand-use.

Reply:
Sure, will fix.